.gitignore

```diff
@@ -7,7 +7,7 @@ yarn-error.log
 vendor/
 
 # Output directories
-build/
+build/bin
```

Comment:
folder `build/windows` will be generated when we build it , so I don't think we should check it in

Reply:
Fixed